tools/retry_autopkgtest.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/sh
+#
+# Retry the autopkgtest run in a pull request.
+# Arguments:
+#   release:      The name of the Ubuntu release to test.
+#   architecture: The architecture to test.
+#   pr_id:        The identifier of the pull request to test.
+# Environment variables:
+#   SNAPCRAFT_AUTOPKGTEST_SECRET: The secret to authenticate the test execution.
+
+if [ -z "${SNAPCRAFT_AUTOPKGTEST_SECRET}" ]; then
+    echo 'Set the secret to the environment variable $SNAPCRAFT_AUTOPKGTEST_SECRET.'
+    exit 1
+fi
+
+if [ "$#" != 3 ]; then
+    echo "Usage: "$0" release architecture pr_id"
```

Comment:
`$0 <release> <architecture> <PR_ID>`

Just `<PR>` would be better, I was thinking if I had to use something different than the number. `<PR#>` if you don't like `<PR>`